origin-dapp/public/index.html

```diff
@@ -47,6 +47,10 @@
       /*eslint-enable */
     </script>
     <!-- End Crowdin -->
+    <!-- Javascript-Load-Image -->
+    <script async src="https://cdnjs.cloudflare.com/ajax/libs/blueimp-load-image/2.20.1/load-image.all.min.js"></script>
```

Comment:
Not sure how we feel about loading a dependency this way--this is different from all others.

Should we consider including it directly, or creating an npm package wrapper for it? 

@jordajm ?

Reply:
FYI @Natasha08 @jordajm  , just learned about *SubResource Integrity* this morning, which enables you to prevent malicious changes to embedded javascript:
https://shkspr.mobi/blog/2018/11/major-sites-running-unauthenticated-javascript-on-their-payment-pages/

(We still should use the npm package, but just sharing this as it was new to me.)